src/main/java/http/MIME.java

```diff
@@ -0,0 +1,42 @@
+package http;
+
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+public enum MIME {
+	HTML("text/html", "html"),
+	CSS("text/css", "css"),
+	JS("application/javascript", "js"),
+	ICO("image/x-icon", "ico"),
+	PNG("image/png", "png"),
+	JPG("image/jpeg", "jpg"),
+	TXT("text/plain", "txt"),
+	EOT("font/EOT", "eot"),
+	SVG("font/SVG", "svg"),
+	WOFF("font/WOFF", "woff"),
+	WOFF2("font/WOFF2", "woff2"),
+	TTF("font/ttf", "ttf");
+
+	public final String contentType;
+	public final String extension;
+
+	MIME(String contentType, String extension) {
+		this.contentType = contentType;
+		this.extension = extension;
+	}
+
+	private String getExtension() {
+		return extension;
+	}
+
+	public boolean isTemplate() {
+		return this.equals(MIME.HTML) || this.equals(MIME.TXT);
+	}
+
+	public static MIME findMimeFromExtension(String extension) {
+		return Arrays.stream(values())
+			.filter(mime -> mime.getExtension().equals(extension))
```

Comment:
같은 클래스 내부에서는 인스터스 변수에 직접 접근할 수 있어요! 🙂

```suggestion
			.filter(mime -> mime.extension.equals(extension))
```